### PR TITLE
Adding a goembed config to simply agent configuration 

### PR DESCRIPTION
**Pull request by khyberspache**
Moves agent configuration to a standalone json file in the `util/conf/default.json`. 
Simplifies the creation of the agent configuration by directly loading the JSON

### CODE REVIEW

**Comment by khyberspache** [comment]
@lzimm We would need to make the Update agent config modal return the AESKey as a string

File: util/config.go [hunk 1]
```diff
@@ -79,25 +84,18 @@ type Instruction struct {
...
-		Useragent: "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.66 Safari/537.36",
-		Sleep:     60,
-		KillSleep: 5,
-		CommandTimeout: 60,
-		Pid: os.Getpid(),
-		Proxy: "",
-		Executing: make(map[string]Instruction),
-	}
+	var agent AgentConfig
+	data, _ := defaultConfig.ReadFile("conf/default.json")
```
Comment (privateducky) [inline util/config.go:88]: Have you tested this after it's been compiled? Go can be finicky about compiling/reading files. I assume that's the reason for the embed lib in this case.

**Review by privateducky — COMMENTED** [review]
One question

File: util/config.go (hunk 1 above)
Comment (khyberspache) [reply util/config.go:88]: Yep! Go Embed is a new feature supported in 1.16+ that allows you to statically compile in file systems. So you could build a HTTP application with a templates folder directly compiled in

**Review by privateducky — APPROVED** [review]
Merge when ready

**Pull request merged**